app/frontend/src/app/app.component.ts

```diff
@@ -26,6 +26,8 @@ export class AppComponent implements OnInit {
     this.log.info("Browser version: ", navigator.userAgent);
     this.log.info("Started PSA frontend Angular application");
 
+    // We don't have an id token, but its required to perform the logout properly, so we just provide a uuid
+    sessionStorage.setItem("id_token", "8c1f517c-9e6d-4881-9a79-8d2ddf8f7c84");
```

Comment:
As the logout is performed by the `token/revoke` request, this is not necessary.